src/containers/now-playing/components/ActionsBar.tsx

```diff
@@ -47,7 +48,8 @@ const ActionsBar = ({
   onToggleFavorite,
   onShare,
   onClickOverflow,
-  isDarkMode
+  isDarkMode,
+  isMatrix
```

Comment:
maybe we should name these isMatrixMode for consistency / it'll be easier to find/replace is.*Mode later?

Reply:
yeah I'm game